tests/components/comelit/test_alarm_control_panel.py

```diff
@@ -0,0 +1,155 @@
+"""Tests for Comelit SimpleHome alarm control panel platform."""
+
+from unittest.mock import AsyncMock
+
+from aiocomelit.api import AlarmDataObject, ComelitVedoAreaObject, ComelitVedoZoneObject
+from aiocomelit.const import AlarmAreaState, AlarmZoneState
+from freezegun.api import FrozenDateTimeFactory
+import pytest
+
+from homeassistant.components.alarm_control_panel import (
+    ATTR_CODE,
+    DOMAIN as ALARM_DOMAIN,
+    SERVICE_ALARM_ARM_AWAY,
+    SERVICE_ALARM_ARM_HOME,
+    SERVICE_ALARM_ARM_NIGHT,
+    SERVICE_ALARM_DISARM,
+    AlarmControlPanelState,
+)
+from homeassistant.components.comelit.const import SCAN_INTERVAL
+from homeassistant.const import ATTR_ENTITY_ID, STATE_UNAVAILABLE
+from homeassistant.core import HomeAssistant
+
+from . import setup_integration
+from .const import VEDO_PIN
+
+from tests.common import MockConfigEntry, async_fire_time_changed
+
+ENTITY_ID = "alarm_control_panel.area0"
+
+
+@pytest.mark.parametrize(
+    ("human_status", "armed", "alarm_state"),
+    [
+        (AlarmAreaState.DISARMED, 0, AlarmControlPanelState.DISARMED),
+        (AlarmAreaState.ARMED, 1, AlarmControlPanelState.ARMED_HOME),
+        (AlarmAreaState.ARMED, 2, AlarmControlPanelState.ARMED_HOME),
+        (AlarmAreaState.ARMED, 3, AlarmControlPanelState.ARMED_NIGHT),
+        (AlarmAreaState.ARMED, 4, AlarmControlPanelState.ARMED_AWAY),
+        (AlarmAreaState.UNKNOWN, 0, STATE_UNAVAILABLE),
+    ],
+)
+async def test_entity_availability(
+    hass: HomeAssistant,
+    freezer: FrozenDateTimeFactory,
+    mock_vedo: AsyncMock,
+    mock_vedo_config_entry: MockConfigEntry,
+    human_status: AlarmAreaState,
+    armed: int,
+    alarm_state: AlarmControlPanelState,
+) -> None:
+    """Test all entities."""
```

Comment:
All of the tests has the same description `"""Test all entities."""` I guess this is a typo?

Reply:
Indeed, fixed